docs/source/cli.rst

```diff
@@ -0,0 +1,56 @@
+OpenAlchemy CLI
+===============
+
+openalchemy build
+-----------------
+
+Description
+^^^^^^^^^^^
+
+Build a Python package containing the models described in the OpenAPI
+specification file.
+
+Usage
+^^^^^
+
+.. program:: openalchemy
+
+.. option:: openalchemy build [OPTIONS] SPECFILE PKG_NAME OUTPUT_DIR
```

Comment:
I think you just need to add `SPECFILE` to the `cspell.json` file under `// Library` on line 3.